docs/user/index.adoc

```diff
@@ -191,28 +191,28 @@ include::../../tests/unit/test_readme_pipelines_are_sequences.py[]
 include::../../tests/unit/comparison_files/test_readme_pipelines_are_sequences_test.yml[]
 ----
 
-== Namespaces allow reuse of jobs and sequences
+== Stages allow reuse of jobs and sequences
```

Comment:
```suggestion
== Stages allow reuse jobs and sequences
```

Reply:
No, that's fine: "Stages allow reuse of what? Of jobs and sequences". Else it should be written "Stages allow to reuse jobs and sequences."